--- banquise.py	2010-12-29 21:40:57.270842267 +0100
+++ banquise.centos.py	2010-12-29 21:40:31.377842267 +0100
@@ -244,8 +244,9 @@
                 "already linked to a valid contract!"
         exit_client()
     config.set("DEFAULT", "uuid", xml)
-    with open("/etc/banquise.conf","wb") as configfile:
-        config.write(configfile)
+    f=open("/etc/banquise.conf","wb")
+    config.write(f)
+    f.close()
 
 def get_release():
     if os.path.exists('/usr/bin/lsb_release'):
